cmd/halt.go

```diff
@@ -0,0 +1,84 @@
+package cmd
+
+import (
+	"flag"
+	"fmt"
+	"strings"
+
+	"trellis-cli/trellis"
+
+	"github.com/mitchellh/cli"
+)
+
+type HaltCommand struct {
+	UI          cli.Ui
+	Trellis     *trellis.Trellis
+	flags       *flag.FlagSet
+	noGalaxy    bool
+	noProvision bool
+}
+
+func NewHaltCommand(ui cli.Ui, trellis *trellis.Trellis) *HaltCommand {
```

Comment:
Since there's no `init` function needed, you can remove this function entirely too